tooling/minimize/tests/pass/enums.rs

```diff
@@ -0,0 +1,112 @@
+fn assert(b: bool) {
+    match b {
+        // FIXME: once we support panics use the safe macro.
+        false => unsafe { std::hint::unreachable_unchecked() },
+        true => {}
+    }
+}
+
+fn test_int_cast() {
+    #[repr(i16)]
+    enum ReprEnum {
+        V1 = -12,
+        V2 = -42,
+    }
+    
+    fn convert_repr_enum(e: ReprEnum) -> i16 {
+        e as i16
+    }
+
+    assert(convert_repr_enum(ReprEnum::V1) == -12);
+    assert(convert_repr_enum(ReprEnum::V2) == -42);
+}
+
+fn test_overaligned_int_cast() {
+    #[repr(align(8))]
+    enum Aligned {
+        Zero = 0,
+        One = 1,
+    }
+
+    assert(Aligned::Zero as u8 == 0);
+    assert(Aligned::One as u8 == 1);
+}
+
+#[allow(unused)]
+fn test_recursive_enum() {
```

Comment:
This isn't actually recursive, just big.

So what happens currently when you enable this?

Reply:
see this: https://github.com/minirust/minirust/issues/161

Reply:
Okay, makes sense.

But please rename the test to `test_big_enum` or so.

Reply:
okay